lib/std.dl

```diff
@@ -77,29 +77,50 @@ extern function str_to_lower(s: string): string
 extern function hash64(x: 'X): bit<64>
 extern function hash128(x: 'X): bit<128>
 
+/* The `Group` type is used exclusively in aggregation operations.  It
+ * represents a non-empty list of objects sorted in ascending order.
```

Comment:
Do you want to enforce the fact that the group is sorted?
Maybe you will want to change your mind later.
At least in LINQ IGroup is not sorted.

Reply:
The sortedness is enforced by differential, not me. This may theoreticlly change in the future, breaking people's code. OTOH, an aggregation algorithm that requires a sorted group may be able to take advantage of this.